src/content/release-notes.mdx

```diff
@@ -5,6 +5,44 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing [Cloud Credentials](admin/cloud-credentials/index.mdx), a central location to manage your cloud provider credentials for `deploy`, `destroy`, and `test` remote operations  <!-- 8550 -->
+- Adding support for [resource quotas at the individual Namespace level](core/namespaces.mdx#configure-namespace-quotas). This allows administrators to set the maximum resources that can be used per Namespace
+- Enabling [Okteto Insights Dashboards](core/okteto-insights-dashboards.mdx) for all SaaS and BYOC users of Okteto <!-- 8627 -->
+- Added CLI, Chart, and Okteto App version to our API headers  <!-- 8545 -->
+- Added support to [enable/disable resource quotas for individual Namespaces](self-hosted/helm-configuration.mdx#quotas). The new default values do not enforce a resource limit  <!-- 8604 -->
```

Comment:
@pchico83 Is this the same than `Adding support for [resource quotas at the individual Namespace level](core/namespaces.mdx#configure-namespace-quotas). This allows administrators to set the maximum resources that can be used per Namespace`? or 2 different things? I mean, we already had quotas for individual namespaces and we only allowed to disable them for some resources, right?

Reply:
It's two different things, but I would unify them in a single feature:
- One feature is that we show the quotas on the Namespace UI. We already did it before, but they have been redesigned
- Another one is that, when you enable `quotas.resource.enabled`, each resource quota cana be enable/disabled independently